### PR TITLE
refactor(3080260634): migrate utils to commons-js

### PR DESCRIPTION
**Pull request by dan-ziv**
### Description of the Changes

Applied all generic utils to the commons repo.

Solves #[3080260634](https://starkware.monday.com/boards/2847524005/pulses/3080260634)

---

### Checklist

- [x] Manually tests of the main Application flows are done and passed.
- [ ] New unit / functional tests have been added (whenever applicable).
- [ ] Docs have been updated (whenever relevant).
- [x] PR title is follow the [Conventional Commits](https://www.conventionalcommits.org/) convention: `<type>[optional scope]: <description>`, e.g: `fix: prevent racing of requests`

- - -
This change is [Reviewable](https://reviewable.io/reviews/starkware-libs/starkgate-frontend/300)

### CODE REVIEW

File: src/components/UI/ChainSelect/ChainSelect.js [hunk 1]
```diff
@@ -1,12 +1,13 @@
 import {Select, MenuItem, FormControl} from '@mui/material';
 import {ChainInfo, ChainType} from '@starkware-industries/commons-js-enums';
+import {openInNewTab} from '@starkware-industries/commons-js-utils';
```
Comment (galuzan-starkware) [inline src/components/UI/ChainSelect/ChainSelect.js:3]: Why 2 lines of imports from the same library?

Comment (dan-ziv) [reply src/components/UI/ChainSelect/ChainSelect.js]: Bad merging, fixed.

File: src/components/UI/LinkButton/LinkButton.js [hunk 2]
```diff
@@ -1,8 +1,9 @@
+import {openInNewTab} from '@starkware-industries/commons-js-utils';
+import {toClasses} from '@starkware-industries/commons-js-utils';
```
Comment (galuzan-starkware) [inline src/components/UI/LinkButton/LinkButton.js:2]: 2 imports from the same libary

File: src/components/UI/Modal/TransactionSubmittedModal/TransactionSubmittedModalButton.js [hunk 3]
```diff
@@ -1,3 +1,5 @@
+import {openInNewTab} from '@starkware-industries/commons-js-utils';
+import {evaluate} from '@starkware-industries/commons-js-utils';
```
Comment (galuzan-starkware) [inline src/components/UI/Modal/TransactionSubmittedModal/TransactionSubmittedModalButton.js:2]: two imports from the same libary

**Review by galuzan-starkware — DISMISSED** [review]

**Pull request merged**